### PR TITLE
Text and link changes for "main" branch

### PR DESCRIPTION
**Pull request by ericherman**
-----
[View rendered GOVERNANCE.md](https://github.com/publiccodenet/about/blob/eh-main-text-links/GOVERNANCE.md)
[View rendered README.md](https://github.com/publiccodenet/about/blob/eh-main-text-links/README.md)
[View rendered activities/codebase-stewardship/product-assets-for-early-incubation.md](https://github.com/publiccodenet/about/blob/eh-main-text-links/activities/codebase-stewardship/product-assets-for-early-incubation.md)
[View rendered activities/communication/blogging.md](https://github.com/publiccodenet/about/blob/eh-main-text-links/activities/communication/blogging.md)
[View rendered activities/communication/get-new-illustration.md](https://github.com/publiccodenet/about/blob/eh-main-text-links/activities/communication/get-new-illustration.md)
[View rendered activities/documentation/index.md](https://github.com/publiccodenet/about/blob/eh-main-text-links/activities/documentation/index.md)
[View rendered activities/documentation/merge-develop-into-master.md](https://github.com/publiccodenet/about/blob/eh-main-text-links/activities/documentation/merge-develop-into-master.md)
[View rendered contributor-guides/for-individuals.md](https://github.com/publiccodenet/about/blob/eh-main-text-links/contributor-guides/for-individuals.md)
[View rendered organization/privacy.md](https://github.com/publiccodenet/about/blob/eh-main-text-links/organization/privacy.md)

### CODE REVIEW

**Comment by ericherman** [comment]
Fixes:

```
- ./_site/organization/privacy.html
  *  External link https://github.com/publiccodenet/about/commits/master/organization/privacy.md failed: 404 No error
```

File: activities/communication/blogging.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ Tag the communications coordinator plus any other colleagues whose review you'd
 
 Once approved, merge your pull request.
 
-To publish the new content, you'll have to create a release branch and [merge develop into master](../documentation/merge-develop-into-master.md).
+To publish the new content, you'll have to create a release branch and [merge develop into main](../documentation/merge-develop-into-master.md).
```
Comment (Ainali) [inline activities/communication/blogging.md:96]: Should we also rename the file it links to?

**Review by Ainali — DISMISSED** [review]
We could possibly also rename activities/documentation/merge-develop-into-master.md  but that is not a blocker.

File: activities/communication/blogging.md (hunk 1 above)
Comment (ericherman) [reply activities/communication/blogging.md]: See follow-on commit.

**Review by Ainali — APPROVED** [review]

**Review by bvhme — APPROVED** [review]

**Pull request merged**